src/Servers/IIS/AspNetCoreModuleV2/OutOfProcessRequestHandler/serverprocess.cpp

```diff
@@ -1118,6 +1125,52 @@ SERVER_PROCESS::SetupStdHandles(
     return hr;
 }
 
+
+void
+SERVER_PROCESS::ReadStdErrHandle(
+    LPVOID pContext
+)
+{
+    auto pLoggingProvider = static_cast<SERVER_PROCESS*>(pContext);
+    DBG_ASSERT(pLoggingProvider != NULL);
+    pLoggingProvider->ReadStdErrHandleInternal();
+}
+
+void
+SERVER_PROCESS::ReadStdErrHandleInternal()
+{
+    std::string tempBuffer;
+    tempBuffer.resize(4096);
```

Comment:
nit: should be able to pass the initial size in the string constructor and save the initial small buffer allocation.

Reply:
I wasn't able to find a constructor that took in just a size for string.